config/inference_router_config.json

```
@@ -0,0 +1,45 @@
+{
+  "basic": {},
+  "health_check": {
+    "check_frequency": "10s",
+    "consecutive_success": 2,
+    "consecutive_failure": 2
+  },
+  "servers": [
+    {
+      "name": "main-server",
+      "addr": "0.0.0.0:8443,::1:8443",
+      "locations": [
+        "inference",
+        "up"
+      ],
+      "threads": 0,
+      "prometheus_metrics": "/metrics"
+    },
+    {
+      "name": "admin-server",
+      "addr": "0.0.0.0:8080,::1:8080",
```

To listen for local requests on 8080 for both ipv4 and ipv6 we should
 ```suggestion
      "addr": "127.0.0.1:8080,::1:8080",
```

---

Done.